### DISCUSSION TITLE
lazer do not submit score if it start by steam

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#27894] [Q&A] [answered]
I don’t know what this means, but I have started lazer through steam a long time ago, and during this time I found that I could not submit score.

[compressed-logs.zip](https://github.com/ppy/osu/files/14994899/compressed-logs.zip)
![image](https://github.com/ppy/osu/assets/34775378/1b639ff1-fa47-41df-97d3-6c2f54418974)
![image](https://github.com/ppy/osu/assets/34775378/03ed034d-aaa1-4212-8016-c40bf7fbacd7)

**Comment by peppy** [comment]
I start the game via steam and it works fine.

Please check the shortcut, you migth have it set to an old version or something else weird.

**Reply by cdwcgt** [reply]
https://github.com/ppy/osu/assets/34775378/d4219158-c20e-425d-836b-6603239b42c1

the log I export in video
[compressed-logs.zip](https://github.com/ppy/osu/files/14998101/compressed-logs.zip)

**Reply by peppy** [reply, marked as answer]
```
2024-04-16 14:09:15 [verbose]: Failed to validate state: System.UnauthorizedAccessException: Access to the path 'C:\Users\cdwcgt\AppData\Roaming\osu\AuthNative.dll' is denied.
```

Dunno. You probably ran the game or steam as admin thus fucking everything on your system.

**Reply by cdwcgt** [reply]
I researched this issue
I tested the new renderer (vulkan) a few days ago, and lazer crashed, but maybe because it was windows problem, lazer didn't exit completely, leaving a zombie process 
![image](https://github.com/ppy/osu/assets/34775378/5e6288a9-7fec-47f1-9e78-d0fdf9a7079b)


As a result, lazer launched through steam cannot access (write?) this file (its also caused damage to my database)
[1713523442.runtime.log](https://github.com/ppy/osu/files/15039176/1713523442.runtime.log)
I don’t know why this problem only occurs for lazer launched by steam, maybe it's because the zombie process is also started by steam?

I'm sorry that I forgot to check the log when creating the discussion, and I deeply apologize for the inconvenience.

**Reply by BlockHead007** [reply]
I got this realm corrupting bug even without steam when using Vulkan renderer, so its not steam related
